### ISSUE TITLE
Video Quality is half when using Simulcast and Screen Sharing

### ISSUE THREAD

**Issue by DanielMcAssey** [jitsi/lib-jitsi-meet#1703]
*This Issue tracker is only for reporting bugs and tracking code related issues.*

Before posting, please make sure you check community.jitsi.org to see if the same or similar bugs have already been discussed. General questions, installation help, and feature requests can also be posted to community.jitsi.org.

## Description
---
When streaming at full 1080p, the receiver is only receiving half 540p even with full bandwidth available (Local testing). This is with p2p disabled
## Current behavior
---
When streaming a screenshare, on the receive side, the receiver is only receiving half the full resolution. (p2p is disabled)

For example I am screen sharing at 1920x1080, the receiver is receiving a WebRTC stream with frameHeight and frameWidth of exactly half.

## Expected Behavior
---
Should be full quality

## Possible Solution
---

## Steps to reproduce
---

# Environment details
---
p2p Disabled
Custom front-end implemented with lib-jitsi-meet

**Comment by jallamsetty1** [comment]
@DanielMcAssey, Do you have layer suspension enabled on your client ?

**Comment by DanielMcAssey** [comment]
@jallamsetty1 Yes Layer suspension is enabled, however the participant was in the Selected/Pinned array.

**Comment by DanielMcAssey** [comment]
Ah okay, no I am not adding to the onStageEndpoints, I assume setReceiverConstraints is the only way to do this? 

I will do this and report back

**Comment by 4n1m4t10n** [comment]
I am having the exact same issue. Since it has been a couple years, I am assuming onStageEndpoints is now onStageSources used in setReceiverConstraints?

**Comment by 4n1m4t10n** [comment]
@jallamsetty1 Is the screenshare endpoint ID used in the receiver constraints the Source name or the track ID?
